src/preload.py

```diff
@@ -51,7 +51,7 @@
     "flash-boot",  # flasher images
 ]
 
-SUPERVISOR_REPOSITORY_RE = "^resin(playground)?/[a-z0-9]+-supervisor$"
+SUPERVISOR_REPOSITORY_RE = "^balena(playground)?/[a-z0-9]+-supervisor$"
```

Comment:
Do we need to match ***either*** `resin` or `balena` instead of just one of them?
The regular expression is matched against this output:
```
            output = docker(
                "images",
                "--all",
                "--format",
                "{{.Repository}} {{.Tag}}"
            )
```
https://github.com/balena-io-modules/balena-preload/blob/v10.4.7/src/preload.py#L754

If the `docker images` command above was executed on an image of an old-enough version of balenaOS, would it contain `resin` instead of `balena`?

Reply:
You're right, versions before v2.26 will still use resin. Also, as we have moved to using the internal registry we will lose the "balena" or "resin" part in the near future, so the regex should be:
```
^(balena|resin)?(playground)?/?[a-z0-9]+-supervisor$
```